backend/load_test/script/quick_test.sh

```diff
@@ -0,0 +1 @@
+artillery quick --count 20 --num 10 http://172.17.0.2:3000/
```

Comment:
Same here, what's the ip address? Shouldn't this be in the .env instead and used as a variable?

Reply:
Not in the scope of the PR, same answer of config.yml